File: leetcode.com/2027_minimum_moves_to_convert_string/solution.py
class Solution:
    def minimumMoves(self, s: str) -> int:
        moves = 0
        a = list(s)

        while 'X' in a:
            for j in range(min(a.index('X') + 3, len(s))):
                a[j] = 'O'

            moves += 1

        return moves

File: leetcode.com/2027_minimum_moves_to_convert_string/solution.c
int minimumMoves(char *s)
{
    int moves = 0;

    for (int i = 0 ; i < strlen(s) ; i++) {
        if (s[i] == 'O')
            continue;

        moves++;
        i += 2;
    }

    return moves;
}
